src/modules/src/esp_deck_flasher.c

```diff
@@ -99,7 +99,7 @@ bool espDeckFlasherWrite(const uint32_t memAddr, const uint8_t writeLen, const u
   // send buffer if full
   const bool sendBufferFull = (sendBufferIndex == ESP_MTU);
   const bool lastPacket = (sequenceNumber == numberOfDataPackets - 1);
-  bool lastPacketFull = false;
+  bool lastPacketFull = lastPacket && (sendBufferIndex == ESP_BITSTREAM_SIZE % ESP_MTU);
```

Comment:
With this the if-statement below is not needed

Reply:
ah oke, thought as it was still up in the air, that it was better just to keep it there to sbe sure. but I'll remove it